src/plugins/telemetryTable/TelemetryTableConfiguration.js

```diff
@@ -38,7 +38,9 @@ define([
             //Make copy of configuration, otherwise change detection is impossible if shared instance is being modified.
             this.oldConfiguration = JSON.parse(JSON.stringify(this.getConfiguration()));
 
-            this.unlistenFromMutation = openmct.objects.observe(domainObject, '*', this.objectMutated);
+            if (domainObject.isMutable !== true) {
```

Comment:
I don't think we need the * listener at all. If it's not mutable, it can't be mutated, so we don't need to listen for any changes.

Reply:
Done